keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -68,15 +68,17 @@ def __init__(
     def text_to_image(
         self,
         prompt,
+        negative_prompt=None,
         batch_size=1,
-        num_steps=25,
+        num_steps=50,
```

Comment:
do you need to change this argument in this PR? Or `num_steps` is by default 50 for `negative_prompt` only?

Reply:
Hi!

It was just to comply with the method `generate_image` that also receives 50 sets as input. I'm not sure whether increasing the number of steps is directly correlated with the negative prompt (and haven't heard anything about it).

I was testing the implementation to see if I didn't break anything and when I did:

```python
from keras_cv.models import StableDiffusion

prompt = "Tacos at dawn"

# Generate step by step
model = StableDiffusion(img_height=512, img_width=512, jit_compile=True)
encoded_text = model.encode_text(prompt)
img = model.generate_image(encoded_text)

# Generate with `text_to_image`
img = model.text_to_image(prompt)
```

Noticed that the two results were different. This is because:
-  `generate_image`: default `num_steps` is 50.
- `text_to_image`: default `num_steps` is 25.

If you want to keep it like this I can revert the change.

Reply:
+1 let's update it to 50 -- this was an oversight that we left this at 25

Reply:
Discussed offline, we should just make it 50 instead